cmd/lsif-server/main.go

```diff
@@ -1,45 +1,103 @@
 package main
 
 import (
+	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
+	"path"
 	"strconv"
 	"strings"
 
+	"github.com/sourcegraph/sourcegraph/internal/env"
 	"github.com/sourcegraph/sourcegraph/internal/goreman"
 )
 
+var (
+	servers = env.Get("LSIF_NUM_SERVERS", "1", "the number of server instances to run (defaults to one)")
+	workers = env.Get("LSIF_NUM_WORKERS", "1", "the number of worker instances to run (defaults to one)")
+
+	// Set in docker image
+	prometheusStorageDir       = os.Getenv("PROMETHEUS_STORAGE_DIR")
+	prometheusConfigurationDir = os.Getenv("PROMETHEUS_CONFIGURATION_DIR")
+)
+
+const (
+	serverPort = 3186
+	workerPort = 3187
+)
+
 func main() {
-	targets, ok := os.LookupEnv("LSIF_RUN_TARGET")
-	if !ok {
-		targets = "server,worker"
+	numServers, err := strconv.ParseInt(servers, 10, 64)
+	if err != nil || numServers < 0 || numServers > 1 {
+		log.Fatalf("invalid int %q for LSIF_NUM_SERVERS: %s", servers, err)
```

Comment:
```suggestion
		log.Fatalf("Invalid int %q for LSIF_NUM_SERVERS: %s", servers, err)
```

If this was in response to https://github.com/golang/go/wiki/CodeReviewComments#error-strings, that guidance doesn't extend to log messages.